gensim/test/test_phrases.py

```diff
@@ -646,6 +646,19 @@ def testEncoding(self):
         self.assertTrue(isinstance(transformed, six.text_type))
 
 
+class TestPhraserModelCompatibilty(unittest.TestCase):
+
+    def TestCompatibilty(self):
+        with temporary_file("phraser_model_3dot6") as fpath:
```

Comment:
This isn't a temporary file. You should use `datapath` for retrieve path to model.